export const errors = {
    en: {
        "noSuchAccountExist": "No such account exist.",
        "invalidMongoId": "{{key}} must be a valid mongo ID",
        "noDatafound": "No Record Found",
        "alreadyExist": "An account with given phoneNumber or email already exists",
        "Exists": "Already Exists! Please try again",
        "userExist": "User already Exist",
        "somethingwrong": "Something went wrong",
        
    }
}

export const success = {
    en: {
        "signupSuccessful": "Signup sucessfully",
        "loginSuccessful": "Login sucessfully",
        "accountExists": "Account already exist",
        "noSuchAccountExist": "No such account exist",
        "respondSuccess": "You have responded successfully",
        "RecordFound": "Record found",
        "issueReported": "issue Reported Successfully",
        "logOutSuccessful": "User logOut successfully",
        "userComment": "User Comments successfully",
        "accountCheckSuccessful": "Account Checked Successfully",
        "otpSendSuccessful": "Otp Send Successfully",
        "otpverifysuccessful": "Otp Verified Successfully",
        "succssDefault": "Success",
        "recordFetched": "Record fetched sucessfully",

    }
}
